nat-lab/tests/conftest.py

```diff
@@ -159,6 +160,35 @@ async def perform_setup_checks() -> bool:
     return True
 
 
+async def kill_natlab_processes():
+    # Do not execute cleanup in non-CI environment,
+    # because cleanup requires elevated privileges,
+    # and it is rather inconvenient when every run
+    # requires to enter sudo password
+    # If someone really wants cleanup - one can set
+    # ENABLE_NATLAB_PROCESS_CLEANUP envron variable
+    if (
+        not "GITLAB_CI" in os.environ
+        and not "ENABLE_NATLAB_PROCESS_CLEANUP" in os.environ
+    ):
+        return
+
+    cleanup_script_path = os.path.join(
+        os.path.dirname(__file__), "../bin/cleanup_natlab_processes"
```

Comment:
For the purpose of notice - it is path traversal, but so strict and environment is so restricted that I don't see window of opportunity here.

Reply:
I do not see path traversal here, there is no user-controllable input (well any other user than the one who is running the script itself). Could you elaborate?

Reply:
fdsa